src/main/java/thePackmaster/patches/DisableRelicStatsForChinesePatch.java

```diff
@@ -0,0 +1,20 @@
+package thePackmaster.patches;
+
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePrefixPatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpireReturn;
+import com.megacrit.cardcrawl.core.Settings;
+import thePackmaster.SpireAnniversary5Mod;
+
+// This exists because relic stats were added after the mod was translated to Chinese (ZHS), so the entries in the
```

Comment:
It's probably worth adding support for this, but I wanted to have this PR entirely functional without needing to coordinate ZHS entries. 

Once this is merged we can see if mwalls is up for adding support for relic stats in ZHS or not. It's mostly just working out where to add the new strings.